### ISSUE TITLE
Should add Mp4/Mkv to Mp3 converter?

### ISSUE THREAD

**Issue by noncapturesen** [fabiogra/moseca#11]
Need to add Video converter in google colab because most of karaoke MTV come with Mp4 or Mkv sir..
please..add ...
OR 
should support Video in Moseca sir

**Comment by Gishi1** [comment]
I think a lossless codec would be better and help prevent distortions in vocals.

**Comment by noncapturesen** [comment]
yep but some 1080p  Video have CompactDisc (CD Quality) which is middle lossless codec.
